src/common/styles/videoConferenceStyle.ts

```diff
@@ -32,6 +32,12 @@ const styles = `
     width: 100%;
   }
 
+  #sv-video-wrapper iframe.sv-video-frame--top { 
+    top: var(--superviz-offset-bottom);
```

Comment:
You should use offset top when positioning on top. 

```suggestion
    top: var(--superviz-offset-top);
```